components/authz-service/server/v2/rules_property_test.go

```diff
@@ -460,7 +463,108 @@ func TestDeleteRuleProperties(t *testing.T) {
 
 				return markedforDeletion && fullyDeleted
 			},
-			createProjectAndRuleGen,
+			createProjectAndRulesGen,
+		))
+
+	properties.TestingRun(t)
+}
+
+func TestListProjectProperties(t *testing.T) {
```

Comment:
This is a "base" test; in case of a failure it would be simpler to trace here rather than in the next test function. Also, this test is checking all fields of a project, whereas the next test function is only checking rules status. That said, every property-based test like this takes from a few seconds to 10 or 20 seconds due to its multiple iterations, so there is a case for deleting this one.